global.net8.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
As currently put together this will not enforce the 8.0 SDK. This should be fine as we can only end up using this global.json when we have already downloaded the 8.0 SDK, but just want to make sure that we are being explicit about what we are doing.

Reply:
We could probably add `rollForward` and explicitly disallow rolling forward on major and minor version.

Reply:
I copied the sdk setup used in dotnet/runtime release/8.0-staging global.json. It uses version 8.0.101, allowPrerelease: true, and rollForward: major. I think this is the correct setup if we have the sdk set as it will use the latest feature found for 8.0.101, but will continue rolling forward to the closest newer sdk. If we want to block against accidentally updating to 9.*, I think using rollForward: "minor" would be correct.

Reply:
With `rollForward: major` we need to make sure there's no .NET 9 SDK around (i.e. globally installed).

Reply:
We could probably also change rollForward to latestMinor since we should be always getting the latest net8.0 version. Let me know what y'all think and I will test it out. (Going to test latestMinor)

Reply:
SGTM.

Reply:
This seems to have worked as expected.